utoipa/src/lib.rs

```diff
@@ -384,6 +384,18 @@ pub trait ToSchema<'__s> {
     }
 }
 
+impl<'__s> ToSchema<'__s> for () {
+    fn schema() -> (&'__s str, openapi::RefOr<openapi::schema::Schema>) {
+        (
+            "UnitType",
+            openapi::schema::ObjectBuilder::new()
+                .nullable(true)
+                .default(Some(serde_json::Value::Null))
+                .into(),
+        )
+    }
+}
+
```

Comment:
I belive this is unnecessary since the implementation is handled in utoipa-gen where the tuple type with zero arguments is being tokenized.

Reply:
Adding this originates in me trying to use `()` in a generic like this:
```rust
type UT = ();
#[derive(ToSchema)]
#[aliases(StrData = Data<UT>)]
pub struct Data<T> {
    #[allow(unused)]
    value: T
}

#[utoipa::path(
    get,
    path = "/root",
    responses (
        ( status = 200, description = "", body = StrData )
    )
)]
pub fn root() -> () {}
```

It's not possible to use `()` in aliases like this `#[aliases(StrData = Data<()>)]`. Therefore I created the type alias `type UT = ();` and used it in `#[aliases(...)]`. With this change the `"value"` property is a reference with `"$ref": "#/components/schemas/UT"`. And for this reference to resolve I need to add `UT` to the schemas `components(schemas(StrData, UT))`.

Reply:
Okay